src/window_gui.h

```diff
@@ -65,6 +66,151 @@ struct WidgetDimensions {
 
 	static const WidgetDimensions unscaled; ///< Unscaled widget dimensions.
 	static WidgetDimensions scaled;         ///< Widget dimensions scaled for current zoom level.
+
+private:
+	/**
+	 * Distances used in drawing widgets.
+	 * These constants should not be used elsewhere, use scaled/unscaled WidgetDimensions instead.
+	 */
+	enum WidgetDrawDistances {
+		/* WWT_IMGBTN(_2) */
+		WD_IMGBTN_LEFT    = 1,      ///< Left offset of the image in the button.
```

Comment:
Now the `WD_` prefix is a bit unneeded. Maybe not for this PR :)

Reply:
To be honest most of these can be removed anyway and replaced with literal numbers in the unscaled definitions. Like I've been doing for new entries.

Reply:
Hmm... looks kinda random... https://gist.github.com/PeterN/d083990f8445f71b4828974b48f80fd0

Reply:
I do like https://gist.github.com/PeterN/d083990f8445f71b4828974b48f80fd0?permalink_comment_id=4834397#gistcomment-4834397. It's quite self explanatory.
Except that left/right/top/bottom is not directly clear. Maybe those could be added for completeness too? But that might clutter it a lot too, so... don't really know.